modules/ROOT/pages/installation/windows.adoc

```diff
@@ -227,3 +227,33 @@ Invoke-Neo4jAdmin
 === Common PowerShell parameters
 
 The module commands support the common PowerShell parameter of `Verbose`.
+
+== Uninstall Neo4j
+
+In case you want to specifically remove Neo4j Desktop, refer to https://neo4j.com/docs/desktop-manual/current/troubleshooting/re-install/[Neo4j Desktop -> Re-installation of Desktop].
+
+Here are the steps to uninstall Neo4j on Windows:
+
+. (Optional) Create a xref:/backup-restore/index.adoc[backup] to avoid losing your data.
+. Stop all Neo4j processes by using the Task Manager.
+. If you have Neo4j Desktop installed, uninstall it via _Control Panel -> Programs_.
```

Comment:
you have instructions here about uninstalling neo4j desktop *and* a link to some other page about how to uninstall neo4j desktop. It seems contradictory to have both.

Earlier this page are instructions to install using zip. So the uninstall is the reverse of that, including first stopping any running services with `bin\neo4j windows-service uninstall`.